tests/bin/get_att.c

```diff
@@ -0,0 +1,18 @@
+#include "libc.h"
+
+int main(void) {
+    // TODO: Good buffer length?
+    unsigned char nonce[512];
+    unsigned char buf[512];
+    size_t technology;
+
+    void* nonce_ptr = &nonce;
+    void* buf_ptr = &buf;
+    size_t* tech_ptr = &technology;
```

Comment:
No need for intermediate variables here.

Reply:
Also note, `nonce` and `buf` are already pointers to buffers. That's how C arrays work.

Reply:
You make a very good point.

Reply:
Drop `tech_ptr`. It isn't needed.